tests/code/test_search.py

```diff
@@ -219,3 +219,18 @@ def test_getDepartmentsForSupervisors():
         assert len(departments) == 3
 
         transaction.rollback()
+
+@pytest.mark.integration
+def test_getSupervisorsForDepartment():
+    with mainDB.atomic() as transaction:
+
+        SupervisorDepartment.create(supervisor = 'B00763721', department = 1) 
+        SupervisorDepartment.create(supervisor = 'B12361006', department = 1)
+
+        supervisors = getSupervisorsForDepartment(1)
+        assert len(supervisors) == 2
+    
+        transaction.rollback()
+    
+    supervisors = getSupervisorsForDepartment(1)
+    assert len(supervisors) == 0
```

Comment:
Consider removing this assertion that department 1 has no supervisors and instead creating a new department that you use for your tests. That way, if someone creates a new entry in our test data doesn't break your test.